src/tests/ftest/erasurecode/online_rebuild_mdtest.yaml

```diff
@@ -58,4 +58,5 @@ mdtest:
       #- [EC_Object_Class, Minimum number of servers]
       - ["EC_2P2GX", 6]
       - ["EC_4P2GX", 8]
+      - ["EC_4P3GX", 9]
```

Comment:
The comment is also wrong here - it uses "exact", so should be `9` -> `12`
https://github.com/daos-stack/daos/blob/7d3241b4933e0696b8fc796909c26f07b0ca6618/src/tests/ftest/erasurecode/online_rebuild_mdtest.py#L43-L45